libtelemetry/src/androidTestFull/java/com/mapbox/android/telemetry/AlarmMangerInstrumentationTest.java

```diff
@@ -18,6 +20,7 @@
 public class AlarmMangerInstrumentationTest {
 
   @Test
+  @Ignore
```

Comment:
let's not ignore, `scheduleExact(...)` checks for >= API 19 or `returns false` since it uses `setExact(...)`, we can use `set(...)` for < API 19. Let's also restrict this API for tests, something like 
```
  @VisibleForTesting
  @RestrictTo(RestrictTo.Scope.TESTS)
  boolean scheduleExact(long interval) {
    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.KITKAT) {
      manager.setExact(AlarmManager.ELAPSED_REALTIME, SystemClock.elapsedRealtime() + interval,
          pendingIntent);
    } else {
      manager.set(AlarmManager.ELAPSED_REALTIME, SystemClock.elapsedRealtime() + interval,
          pendingIntent);
    }
    return true;
  }
```

Reply:
Does this guarantee the test to pass? This is because the test asserts the alarm at definite time and documentation for set() method does not talk about accuracy?

Reply:
`set` became `inexact` in API 19 and `setExact` was added so prior to 19 `set` should work.
https://developer.android.com/reference/android/app/AlarmManager#set(int,%20long,%20android.app.PendingIntent)
Docs - `Applications whose targetSdkVersion is before API 19 will continue to get the previous alarm behavior: all of their scheduled alarms will be treated as exact.`